src/main/java/fruitshop/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,23 @@
+package fruitshop.service.impl;
+
+import fruitshop.service.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String file) {
+        List<String> data = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(file))) {
+            String value = bufferedReader.readLine(); // skip header (first line)
```

Comment:
The first line from the file is read and then immediately discarded because of the second read inside the loop condition. If the first line contains headers, this is fine, but if it contains valid data, this would result in losing information.